addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -98,10 +100,9 @@
 <!-- Slide sub-template: display an item in a list of related slides (Related, Most Viewed, ...) -->
 <template id="slide_aside_card" name="Related Slide">
     <a class="list-group-item list-group-item-action d-flex align-items-start px-2" t-att-href="'/slides/slide/%s' % (slug(aside_slide))">
-        <t t-set="slide_image" t-value="website.image_url(aside_slide, 'image_1024')"/>
+        <t t-set="slide_image" t-value="website.image_url(aside_slide, 'image_1024') if can_access_channel or aside_slide.is_preview else default_background_image"/>
 
-        <div t-if="aside_slide.image_1024" class="flex-shrink-0 mr-1 border" t-attf-style="width: 20%; padding-top: 20%; background-image: url(#{slide_image}); background-size: cover; background-position:center"/>
-        <div t-else="" class="o_wslides_card_default_background flex-shrink-0 mr-1" t-attf-style="width: 20%; padding-top: 20%;"/>
+        <div class="mr-1 border" t-attf-style="width: 20%; padding-top: 20%; background-image: url(#{slide_image}); background-size: cover; background-position:center"/>
```

Comment:
Oh indeed I see now what the `flex-shrink-0` did. Hum like you say either we keep the class or we set a min width.
We already use `style` here so I guess we can use `min-width` instread of `width` for this one. Not really critical so I'm fine with what you like.

Reply:
Unless we add `flex-grow` with the `min-width`, the image div will never grow either, so `width` does the same thing, right?
Or you'd like this behavior instead of the square?
![image](https://user-images.githubusercontent.com/90857153/175905790-94685d81-c1d3-4cea-928b-d6b8f47f8ef5.png)
![image](https://user-images.githubusercontent.com/90857153/175906346-4ddeffb7-0075-44c4-b20a-9e9791461c4b.png)

Reply:
Log: yes, min-width was required, thanks for insisting @jeh-odoo. 
(`width` is not enough to disable shrink in flexbox, details here https://stackoverflow.com/questions/21406397/width-ignored-on-flexbox-items)